build.gradle

```diff
@@ -151,13 +151,13 @@ allprojects {
       events 'passed', 'skipped', 'failed'
       showStandardStreams = true
     }
-  }
 
-  tasks.withType(Test).configureEach {
-    systemProperties["junit.jupiter.execution.parallel.enabled"] = true
+    systemProperties["junit.jupiter.execution.parallel.enabled"] =  true
```

Comment:
Let's address it in a separate PR. E.g. [here](https://github.com/Consensys/maru/pull/184)